packages/components/src/internal/components/pipeline/utils.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import { AppURL } from '../../url/AppURL';
+import { App, AssayUploadResultModel } from '../../../index';
+
+export function getPipelineLinkMsg(response: AssayUploadResultModel): JSX.Element {
+    return (
+        <>
+            Click{' '}
+            <a href={AppURL.create('pipeline', response.jobId).toHref()}>  here </a> to check the status of the background
```

Comment:
nit: Why all this additional spacing around `>      here <`?

Reply:
(An accident—removed)